styleguide/components/21-atoms/lists/_list.scss

```diff
@@ -40,9 +40,16 @@ ul {
     }
   }
 
-  &.iconlist-small li::before {
-    font-size: 1.2rem;
-    line-height: 1.7em;
+  &.iconlist-small {
+    li {
+      font-size: .8rem;
+      line-height: 2.4;
+
+      &::before {
+        font-size: 1.2rem;
+        line-height: 1.7em;
```

Comment:
remove 'em', maybe we need to discuss if we wan't an em or not at line-height :) ?

Reply:
We don't, it's two bytes of useless data.

Reply:
Done...